### PR TITLE
Adds debug and admin tools

### PR DESCRIPTION
**Pull request by rsarai**
#119  and #31 

## Description

Adds django-debug-toolbar, django-debug-toolbar-request-history and django-import-export

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: project_name/settings/local_base.py [hunk 1]
```diff
@@ -33,6 +33,27 @@
 EMAIL_BACKEND = 'naomi.mail.backends.naomi.NaomiBackend'
 EMAIL_FILE_PATH = base_dir_join('tmp_email')
 
+# django-debug-toolbar and django-debug-toolbar-request-history
+INSTALLED_APPS += ('debug_toolbar',)
+MIDDLEWARE += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+INTERNAL_IPS = '127.0.0.1'
```
Comment (chocoelho) [inline project_name/settings/local_base.py]: `INTERNAL_IPS` should be a list. I think we should follow a pattern like the one described in this [example settings](https://github.com/jazzband/django-debug-toolbar/blob/e7a9ed0c3de1c8635f7e8c83349cb77cef42489f/example/settings.py#L14).

Comment (chocoelho) [reply project_name/settings/local_base.py]: @rsarai any updates on this?

Comment (rsarai) [reply project_name/settings/local_base.py]: Yeah! Sorry for the delay, you have this done this week!

Comment (lucabezerra) [reply project_name/settings/local_base.py]: @chocoelho I see you've assigned yourself on Oct 20th, but shouldn't this be assigned to @rsarai ? :p

**Comment by chocoelho** [comment]
If she is going to be responsible for merging it, then np :smile:

Em 20 de nov de 2017 00:27, "Luca Bezerra" <notifications@github.com>
escreveu:

**Pull request merged**